src/ansi-c/ansi_c_scope.cpp

```diff
@@ -10,6 +10,11 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "ansi_c_scope.h"
 
+std::ostream &operator<<(std::ostream &os, ansi_c_id_classt c)
+{
+  return os << static_cast<int>(c);
+}
+
```

Comment:
The current behaviour is to print the integer value of the enum. For now I'll use a `static_cast` directly when printing, and a proper `<<` can be added later if it becomes necessary.